db/migrate/20160516070529_divide_lookup_key_permissions.rb

```diff
@@ -1,14 +1,32 @@
 class DivideLookupKeyPermissions < ActiveRecord::Migration
+  class FakeFilter < ActiveRecord::Base
```

Comment:
Some details + related issues to cases, when the models are used in migrations http://projects.theforeman.org/issues/15866